homeassistant/components/econet/__init__.py

```diff
@@ -105,6 +105,7 @@ class EcoNetEntity(Entity):
     """Define a base EcoNet entity."""
 
     _attr_should_poll = False
+    _attr_temperature_unit = UnitOfTemperature.FAHRENHEIT
```

Comment:
`_attr_temperature_unit: str = UnitOfTemperature.FAHRENHEIT` should do the trick for mypy (as seen in `components/matter/climate.py` )

Reply:
I now moved them to the respective classes. having the property in a super class is a bit too unclear imo as you can't really find any trace of the value in the entity itself.